examples/nodejs/testingbot/bot.js

```diff
@@ -0,0 +1,179 @@
+"use strict";
+const {
+    MediaStream,
+    nonstandard: { RTCAudioSink }
+} = require("wrtc");
+const {
+    HiFiAudioAPIData,
+    Point3D,
+    OrientationEuler3D,
+    HiFiCommunicator,
+    HiFiConstants
+} = require("hifi-spatial-audio");
+const { default: SignJWT } = require("jose/jwt/sign");
+const { default: UnsecuredJWT } = require("jose/jwt/unsecured");

```

Comment:
`esm` doesn't play nicely with `jose`. To fix that, you'll have to do:

```suggestion
const { SignJWT } = require('jose/dist/node/cjs/jwt/sign');
const { UnsecuredJWT } = require('jose/dist/node/cjs/jwt/unsecured');
```